Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -3700,10 +3700,15 @@ static NSStringCompareOptions compareOptionsForFindOptions(WebCore::FindOptions
         return { { }, nil };
 
     for (PDFAnnotation *annotation in annotationsForCurrentPage.get()) {
-        if (!annotationIsExternalLink(annotation))
+        FloatRect annotationBoundsInPageSpace = [annotation bounds];
+
+        if (!annotationBoundsInPageSpace.contains(pagePoint))
             continue;
 
-        if (!FloatRect { [annotation bounds] }.contains(pagePoint))
+        if (m_activeAnnotation && m_activeAnnotation->annotation() == annotation)
+            data.isActivePDFAnnotation = true;
```

Comment:
Since we don't initialize `isActivePDFAnnotation` to `false` in the structure declaration, I suppose we should always populate it with a valid boolean, lest the garbage value it's initialized with produces unexpected side effects.

```suggestion
        data.isActivePDFAnnotation = m_activeAnnotation && m_activeAnnotation->annotation() == annotation;
```

Reply:
Actually, we want to `data.isActivePDFAnnotation = false` as soon as we enter `UnifiedPDFPlugin::textForImmediateActionHitTestAtPoint()`

Reply:
Good point! It looks like the constructors for this struct are handling the initialization of its members so I did the same